packages/core/src/item.ts

```diff
@@ -75,7 +75,7 @@ export const FIELD_DEFS: { [t in FieldType]: FieldDef } = {
     },
     [FieldType.Url]: {
         type: FieldType.Url,
-        pattern: ".*",
+        pattern: /[(http(s)?):\/\/(www\.)?a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,8}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/i,
```

Comment:
This won't recognize unicode URLs, but I figured it was a nice improvement anyway.